src/sidebar/components/sidebar-content-error.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const { Fragment, createElement } = require('preact');
+const propTypes = require('prop-types');
+
+/**
+ * An error message to display in the sidebar.
+ */
+function SidebarContentError({
+  loggedOutErrorMessage,
+  loggedInErrorMessage,
+  onLoginRequest,
+  isLoggedIn,
+}) {
+  return (
+    <div className="annotation-unavailable-message">
+      <div className="annotation-unavailable-message__icon" />
+      <p className="annotation-unavailable-message__label">
+        {!isLoggedIn ? (
+          <Fragment>
+            {loggedOutErrorMessage}
+            <br />
+            You may need to{' '}
+            <a
+              className="loggedout-message__link"
+              href=""
+              onClick={onLoginRequest}
+            >
+              log in{' '}
```

Comment:
```suggestion
              log in{' '}
```

The space at the end here should be moved outside of the anchor.

Reply:
Eh... I'm going to merge this anyway so I can look at the other open PR.